retranslate/run.sh

```diff
@@ -0,0 +1,37 @@
+# setup environment
+export NEW_SOURCE_PO=${NEW_SOURCE_PO:-'./public/i18n/en_US.txt'}
+export NODE_ENV=development
+
+# ensure i18n & src folder are reset 
+git checkout -- public/i18n/ src
```

Comment:
I'd be afraid that this script accidentally my changes.

![image](https://user-images.githubusercontent.com/192912/43838137-cc130b48-9b1a-11e8-9292-aab6678ae1f0.png)

How about if this exits if there are changes and lets the user do the cleanup themselves?

Maybe like this?

```bash
test -z "$(git status --porcelain)" || (echo "Your repo must be clean"; exit 1)
```

Reply:
Oh yes, this was more for dev, will add